src/Components/Form/Form.js

```diff
@@ -185,7 +186,8 @@ export class Form extends Component {
   handleSubmission = (event) => {
     event.preventDefault();
     if (this.checkForAllQuestions()) {
```

Comment:
I'm assuming this is the error handling we were talking about earlier, so that the questions shown with the submit button will be checked for relevant answers before the results page is displayed.  Nice work!

Reply:
Yeah! just split it up a bit to make sure it was able to be invoked in multiple places while keeping it DRY